site/composer.json

```diff
@@ -12,7 +12,9 @@
   },
   "require": {
     "twig/twig": "^2.9.0",
-    "ramsey/uuid": "^3.8.0"
+    "ramsey/uuid": "^3.8.0",
+    "aptoma/twig-markdown": "~2.0",
```

Comment:
Why are we using the 2.x and not 3.x for this plugin? `"aptoma/twig--markdown": "^3.1.0"`, `"michelf/php-markdown": "^1.8.0"`

Let's also prefer to use the `^` over `~` unless there's a set reason to not bring in new minor versions.